includes/classes/class-omise-card-image.php

```diff
@@ -0,0 +1,198 @@
+			return self::get_image( 'diners.svg', 'Diners Club' );
+		}
+
+		/**
+		 * Return the default setting of display the JCB logo
+		 * 
+		 * @return string
+		 */
+		public static function get_jcb_default_display() {
+			return 'no';
+		}
+
+		/**
+		 * Return the HTML &lt;img&gt; element of JCB logo
+		 * 
+		 * @return string
+		 */
+		public static function get_jcb_image() {
+			return self::get_image( 'jcb.svg', 'JCB' );
+		}
+
+		/**
+		 * Return the default setting of display the MasterCard logo
+		 *
+		 * @return string
+		 */
+		public static function get_mastercard_default_display() {
+			return 'yes';
+		}
+
+		/**
+		 * Return the HTML &lt;img&gt; element of MasterCard logo
+		 * 
+		 * @return string
+		 */
+		public static function get_mastercard_image() {
+			return self::get_image( 'mastercard.svg', 'MasterCard' );
+		}
+
+		/**
+		 * Return the default setting of display the Visa logo
+		 *
+		 * @return string
+		 */
+		public static function get_visa_default_display() {
+			return 'yes';
+		}
+
+		/**
+		 * Return the HTML &lt;img&gt; element of Visa logo
+		 * 
+		 * @return string
+		 */
+		 public static function get_visa_image() {
+			return self::get_image( 'visa.svg', 'Visa' );
+		}
+
+		/**
+		 * Check whether the setting for American Express logo is configured and it was set to display or not display
+		 * 
+		 * @param mixed $setting The array that contains key for checking the flag
+		 * @return boolean
+		 */
+		public static function is_amex_enabled( $setting ) {
+			if ( isset( $setting['accept_amex'] ) && $setting['accept_amex'] == 'yes' ) {
+				return true;
+			} else {
+				return false;
+			}
+		}
+
+		/**
+		 * Check whether the setting for Diners Club logo is configured and it was set to display or not display
+		 * 
+		 * @param mixed $setting The array that contains key for checking the flag
+		 * @return boolean
+		 */
+		public static function is_diners_enabled( $setting ) {
+			if ( isset( $setting['accept_diners'] ) && $setting['accept_diners'] == 'yes' ) {
+				return true;
+			} else {
+				return false;
+			}
+		}
+
+		/**
+		 * Check whether the setting for JCB logo is configured and it was set to display or not display
+		 * 
+		 * @param mixed $setting The array that contains key for checking the flag
+		 * @return boolean
+		 */
+		public static function is_jcb_enabled( $setting ) {
+			if ( isset( $setting['accept_jcb'] ) && $setting['accept_jcb'] == 'yes' ) {
+				return true;
+			} else {
+				return false;
+			}
+		}
+
+		/**
+		 * Check whether the setting for MasterCard logo is configured and it was set to display or not display
+		 * 
+		 * @param mixed $setting The array that contains key for checking the flag
+		 * @return boolean
+		 */
+		public static function is_mastercard_enabled( $setting ) {
+			// Make it backward compatible. If the setting is not configured, the MasterCard logo is display by default.
+			if ( ! isset( $setting['accept_mastercard'] ) ) {
```

Comment:
For me, it's different in terms of code clean and reduce some nested `if ... else`.
this one also
```php
public static function is_jcb_enabled( $setting ) {
    if ( isset( $setting['accept_jcb'] ) && $setting['accept_jcb'] == 'yes' ) {
        return true;
    } else {
        return false;
    }
}
```

it could be
```php
public static function is_jcb_enabled( $setting ) {
    if ( isset( $setting['accept_jcb'] ) && $setting['accept_jcb'] == 'yes' ) {
        return true;
    }

    return false;
}
```

But yeah, it provides the same result and not much different as you said 😂

👍